pandas/core/array_algos/quantile.py

```diff
@@ -102,7 +104,8 @@ def quantile_with_mask(
             interpolation=interpolation,
         )
 
-        result = np.array(result, copy=False)
+        copy_false = None if np_version_gt2 else False
+        result = np.array(result, copy=copy_false)
```

Comment:
```suggestion
        result = np.asarray(result)
```

Would this be a simpler but equivalent alternative?

Reply:
Looks like it (from a quick glance at the diff).

I just added this to the 2.0 migration guide:

1. Code using ``np.array(..., copy=False)`` can in most cases be changed to
   ``np.asarray(...)``. Older code tended to use ``np.array`` like this because
   it had less overhead than the default ``np.asarray`` copy-if-needed
   behavior. This is no longer true, and ``np.asarray`` is the preferred function.